PhysicsTools/NanoAOD/python/taus_updatedMVAIds_cff.py

```diff
@@ -355,194 +356,83 @@
+)
+## anti-e v1 WPs
+# VLoose
+patTauDiscriminationByVLooseElectronRejectionMVA62015 = patTauDiscriminationByVLooseElectronRejectionMVA62018.clone(
+    PATTauProducer = patTauDiscriminationByElectronRejectionMVA62015Raw.PATTauProducer,
+    Prediscriminants = patTauDiscriminationByElectronRejectionMVA62015Raw.Prediscriminants,
+    toMultiplex = cms.InputTag("patTauDiscriminationByElectronRejectionMVA62015Raw"),
+    key = cms.InputTag("patTauDiscriminationByElectronRejectionMVA62015Raw","category")
 )
+for m in patTauDiscriminationByVLooseElectronRejectionMVA62015.mapping:
+    m.cut = m.cut.value().replace(antiElectronDiscrMVA6_version, antiElectronDiscrMVA6v1_version).replace("WPeff98", "WPEff99")
+# Loose
+patTauDiscriminationByLooseElectronRejectionMVA62015 = patTauDiscriminationByVLooseElectronRejectionMVA62015.clone()
+for m in patTauDiscriminationByLooseElectronRejectionMVA62015.mapping:
+    m.cut = m.cut.value().replace("WPEff99", "WPEff96")
+# Medium
+patTauDiscriminationByMediumElectronRejectionMVA62015 = patTauDiscriminationByVLooseElectronRejectionMVA62015.clone()
+for m in patTauDiscriminationByMediumElectronRejectionMVA62015.mapping:
+    m.cut = m.cut.value().replace("WPEff99", "WPEff91")
+# Tight
+patTauDiscriminationByTightElectronRejectionMVA62015 = patTauDiscriminationByVLooseElectronRejectionMVA62015.clone()
+for m in patTauDiscriminationByTightElectronRejectionMVA62015.mapping:
+    m.cut = m.cut.value().replace("WPEff99", "WPEff85")
+# VTight
+patTauDiscriminationByVTightElectronRejectionMVA62015 = patTauDiscriminationByVLooseElectronRejectionMVA62015.clone()
+for m in patTauDiscriminationByVTightElectronRejectionMVA62015.mapping:
+    m.cut = m.cut.value().replace("WPEff99", "WPEff79")
 ### Put all anti-e tau-IDs into a sequence
-patTauDiscriminationByElectronRejectionSeq = cms.Sequence(
+_patTauDiscriminationByElectronRejection2018Seq = cms.Sequence(
     patTauDiscriminationByElectronRejectionMVA62018Raw
     +patTauDiscriminationByVLooseElectronRejectionMVA62018
     +patTauDiscriminationByLooseElectronRejectionMVA62018
     +patTauDiscriminationByMediumElectronRejectionMVA62018
     +patTauDiscriminationByTightElectronRejectionMVA62018
     +patTauDiscriminationByVTightElectronRejectionMVA62018
 )
+_patTauDiscriminationByElectronRejection2015Seq = cms.Sequence(
+    patTauDiscriminationByElectronRejectionMVA62015Raw
+    +patTauDiscriminationByVLooseElectronRejectionMVA62015
+    +patTauDiscriminationByLooseElectronRejectionMVA62015
+    +patTauDiscriminationByMediumElectronRejectionMVA62015
+    +patTauDiscriminationByTightElectronRejectionMVA62015
+    +patTauDiscriminationByVTightElectronRejectionMVA62015
+)
+patTauDiscriminationByElectronRejectionSeq = _patTauDiscriminationByElectronRejection2018Seq.copy()
+run2_miniAOD_devel.toReplaceWith(patTauDiscriminationByElectronRejectionSeq,
+                                 _patTauDiscriminationByElectronRejection2015Seq)
+
```

Comment:
for the UL reMini this use the 2015Seq while for the old mini the 2018 (?)
this is what you want to do ?
note: this is consistent with the master.

Reply:
It is correct. The 2015Seq is run when miniAOD contains 2018 anti-e tauIDs (as it master and UL reMiniAOD), while 2018Seq when miniAOD contains 2015 anti-e tauIDs (older miniAOD). In this way nanoAOD contains complete set of anti-e tauIDs, i.e. 2015+2018.